PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalInternalClient.kt

```diff
@@ -158,7 +157,14 @@ internal class PayPalInternalClient(
                 )
                 if (getAppSwitchUseCase()) {
                     if (!contextId.isNullOrEmpty()) {
-                        paymentAuthRequest.approvalUrl = createAppSwitchUri(parsedRedirectUri).toString()
+                        val flowType = if (payPalRequest.isBillingAgreement()) VAULT_FLOW_TYPE else ECS_FLOW_TYPE
+                        val merchantId = configuration.merchantId
+
+                        paymentAuthRequest.approvalUrl = createAppSwitchUri(
+                            parsedRedirectUri,
+                            flowType,
+                            merchantId
+                        ).toString()
```

Comment:
```suggestion
                        paymentAuthRequest.approvalUrl = createAppSwitchUri(
                            uri = parsedRedirectUri,
                            flowType = if (payPalRequest.isBillingAgreement()) VAULT_FLOW_TYPE else ECS_FLOW_TYPE,
                            merchantId = configuration.merchantId
                        ).toString()
```

Take it or leave it suggestion - not sure if it reads a little better without storing flowType and merchantId into variables.

Reply:
Taking this suggestion. Thanks.

Reply:
I think in that case we can store flowType to a variable.